floodviz/thumbnail/thumbnail.js

```diff
@@ -82,7 +83,7 @@ function convert(figure, window, css_path, filename) {
 		svg_string = inject_style(style_default, null, svg, window);
 	}
 	// Takes care of canvas conversion and encodes base64
-	svg2png(svg_string)
+	svg2png(svg_string, {height: 300, width: 560})
```

Comment:
There was refactoring done to height and width within the figure module while I was working on implementing the thumbnail script. When jumping on this current ticket, height and width had been removed from hydrograph.js during svg initialization (lines 116-122). This required me to specify height in width in svg2png because it could not be interpreted from the svg itself.